DQMServices/Components/plugins/MEtoEDMConverter.cc

```diff
@@ -92,7 +94,7 @@ void
 MEtoEDMConverter::beginJob()
 {
   // Determine if we are running multithreading asking to the DQMStore. Not to be moved in the ctor
-  enableMultiThread_ = dbe->enableMultiThread_;
+  //enableMultiThread_ = dbe->enableMultiThread_;
```

Comment:
just delete the commented-out lines (and drop the empty methods)

Reply:
This commit says `HACK`  in the message for a good reason. After #22218, `enableMultiThread` has exactly no reason for existence, but I'll delay removing it everywhere to after #22281 is resolved.